### DISCUSSION TITLE
News Cache [2023-10-30]

### DISCUSSION THREAD

**Discussion by amyegan** [vercel/community#4671] [Announcements]
### 🚢  Ships

- [Next.js 14](https://nextjs.org/blog/next-14)
    - [Server Actions](https://nextjs.org/blog/next-14#forms-and-mutations) (Stable)
    - [Partial Prerendering](https://nextjs.org/blog/next-14#partial-prerendering-preview) (Preview)
    - [Metadata Improvements](https://nextjs.org/blog/next-14#metadata-improvements)
    - New [Learn Next.js](https://nextjs.org/learn) course
- [Home page redesign](https://x.com/rauchg/status/1716608678651211878?s=20) ✨
- [Geist](https://vercel.com/font), a free and open-source fount created in collaboration with [basement.studio](https://basement.studio/)

**Comment by JonathanXDR** [comment]
Dear Vercel team and community,

With the recent announcement of Geist, a free and open-source font created in collaboration with Basement Studio, I'm excited about the innovative tools and resources Vercel is developing. This brings me to a question regarding the [Geist Design System](https://vercel.com/design/introduction). A few years ago, there was a discussion ([Discussion #238](https://github.com/orgs/vercel/discussions/238)) indicating plans to make the Geist Design System public. Given the advancements and releases like Geist, I'm curious to know if there are any updates or timelines on when the Geist Design System will be made available to the public? It would be fantastic to have access to such a comprehensive design system for development and design purposes.

Thanks for your continuous efforts and looking forward to any updates you can share!

Best regards,
Jonathan Russ

**Reply by amyegan** [reply]
Hi Jonathan! Thanks for your interest in Geist. Though interactive documentation for our design system is publicly available at [vercel.com/design/introduction](https://vercel.com/design/introduction), we don't have an ETA for releasing a UI component library. 

If you're interested in an existing open source component library with a similar style, [shadcn/ui](https://ui.shadcn.com/) is a great option.

**Reply by JonathanXDR** [reply]
Alright, I will try that then. Thanks for your reply! :D
